ios/RCCNotification.m

```diff
@@ -335,6 +340,11 @@ -(void)performShowWithCompletion:(void (^)(void))completion
 
 -(void)showWithCompletion:(void (^)(void))completion
 {
+    if(self.hideStatusBar){
+        UIWindow *window = [[RCCManager sharedInstance] getAppWindow];
+        window.windowLevel = UIWindowLevelAlert;
```

Comment:
I don't think this is a safe way to implement this, as we have no guarantee about the other window levels... I'd rather see the alert be displayed on a new `UIWindow`, rather than reusing the existing one, to implement this feature.